src/components/shared/footer/footer.jsx

```diff
@@ -29,29 +35,40 @@ const Footer = ({ isDocPage = false, withTopBorder = false, theme = 'white' }) =
             </Link>
             {isDocPage && <ThemeSelect className="mt-10 xl:mt-11 md:mt-0" />}
           </div>
-          <div className="space-y-[18px] leading-none">
+          <div
+            className={clsx(
+              { 'tracking-tight text-pricing-gray-8': theme === 'pricing' },
+              'space-y-[18px] leading-none'
+            )}
+          >
             <p>Made in SF and the World</p>
             <p>Neon 2023 Ⓒ All rights reserved</p>
           </div>
         </div>
         <div className="flex space-x-[123px] xl:space-x-8 md:hidden">
           {MENUS.footer.map(({ heading, links }, index) => (
             <div className={clsx('flex flex-col xl:w-full')} key={index}>
-              <h3 className="relative text-sm font-bold uppercase leading-none tracking-wider">
+              <h3
+                className={clsx(
+                  { 'text-[13px] font-semibold text-pricing-gray-6': theme === 'pricing' },
+                  'relative text-sm font-bold uppercase leading-none tracking-wider'
+                )}
+              >
                 {heading}
               </h3>
               <ul className="mt-6 flex grow flex-col space-y-[18px]">
-                {links.map(({ to, text }, index) => {
+                {links.map(({ to, text, icon: Icon }, index) => {
```

Comment:
What if there are no links?

Reply:
Naturally, there will always be links in the `MENUS` 🙂